draft-saml-ipsie-sl1-profile.md

```diff
@@ -0,0 +1,382 @@
+- shall sign all assertions using XML Signature [XMLDSig];
+- shall encrypt all assertions containing personally identifiable information (PII) or sensitive attributes using XML Encryption [XMLEnc] with AES-GCM with a minimum key length of 128 bits;
+- shall not expose open redirectors;
+- shall only accept entity IDs registered in metadata as audience values;
+- shall issue assertions with a maximum lifetime of 5 minutes;
+- shall require Service Providers to be preregistered, and shall not support dynamic registration of Service Providers;
+- shall support assertion revocation or status verification mechanisms (such as OCSP for certificate checking);
+- shall implement key rotation procedures with a maximum key lifetime of 2 years for signing keys.
+
+SAML Assertions issued by Identity Providers:
+
+- shall contain the Service Provider's entity ID as the audience value;
+- shall contain an AuthnContext element that identifies the authentication context class that the authentication performed satisfied;
+- shall include an AuthnContextClassRef indicating the authentication method used, which must reflect multi-factor authentication per NIST 800-63-4 AAL2 requirements;
+- shall indicate the expected lifetime of the Service Provider session in the SessionNotOnOrAfter attribute;
+- shall include NotBefore and NotOnOrAfter conditions with a maximum validity period of 5 minutes;
+- shall include a unique identifier for the assertion that is never reused;
+- shall include a SubjectConfirmation element that implements holder-of-key binding when assertions contain sensitive information or authorize high-value transactions;
+- shall include attribute statements that use pairwise pseudonymous identifiers for privacy protection when appropriate;
+- shall minimize disclosure of personal information by only including required attributes in assertions.
+
+For the Web Browser SSO Profile, Identity Providers:
+
+- shall support HTTP-Redirect and HTTP-POST bindings for the SSO service;
+- shall generate a unique ID for each assertion;
+- shall include a ResponseID attribute that is unique for each response;
+- shall not transmit SAML responses over unencrypted network connections;
+- shall encrypt assertions using XML Encryption [XMLEnc] with AES-GCM with a minimum key length of 128 bits;
+- shall reject authentication requests that have been previously processed (replay detection);
+- shall not use the HTTP 307 status code when redirecting a request that contains user credentials;
+- shall use the HTTP 303 status code when redirecting the user agent using status codes;
+- shall support RelayState values up to 80 bytes in length, may reject RelayState values longer than 80 bytes;
+- shall require multi-factor authentication for all authentication events and encode this fact in the AuthnContextClassRef element;
+- shall maintain records of federation events for a minimum of 7 years or as required by applicable regulations;
```

Comment:
Data retention requirements for IPSIE seems like a different topic to debate and not linked to levels.